_inc/client/components/jetpack-termination-dialog/features.jsx

```diff
@@ -134,6 +145,29 @@ class JetpackTerminationDialogFeatures extends Component {
 						</ul>
 					</div>
 				) }
+				{ connectedPlugins.length > 0 && (
+					<div className="jetpack-termination-dialog__generic-info">
+						<h2>
+							{ __(
+								'Jetpack Connection is also used by other plugin',
```

Comment:
Maybe this text would be better:

```suggestion
								'The Jetpack Connection is also used by another plugin',
```